src/api/geocode.js

```diff
@@ -92,7 +92,7 @@ export default function geocodeApi() {
         apiKey: HERE_API_KEY,
         q: query,
         limit: 20,
-        details: '1',
+        show: ['details'],
```

Comment:
this was actually undocumented I think? but I found it on stack overflow
https://stackoverflow.com/a/68708644